src/composables/useNetwork.ts

```diff
@@ -11,9 +11,9 @@ const DEFAULT_NETWORK_ID =
 
 export const networkId = ref<Network>(DEFAULT_NETWORK_ID);
 
-export const isMainnet = networkId.value === Network.MAINNET;
-export const isPolygon = networkId.value === Network.POLYGON;
-export const isArbitrum = networkId.value === Network.ARBITRUM;
+export const isMainnet = computed(() => networkId.value === Network.MAINNET);
+export const isPolygon = computed(() => networkId.value === Network.POLYGON);
+export const isArbitrum = computed(() => networkId.value === Network.ARBITRUM);
```

Comment:
Yeah I have checked - its not really used because we mostly get it from `useWeb3`, but in the future its the one we should use for the app will support multiple networks. (since its all going to be based on this `networkId` ref)